### PR TITLE
refactor: delete task sender

### PR DESCRIPTION
**Pull request by NicolasRampoldi**
# Description

- This PR deletes the (in)famous task_sender.

# To Test

- Generate proofs with different proof systems and try different make targets that send the proofs to the batcher.

### CODE REVIEW

File: scripts/test_examples/risc_zero/fibonacci_proof_generator/.gitignore [hunk 1]
```diff
@@ -6,7 +6,3 @@
 
 # MacOS nuisances
 .DS_Store
-
-# Proofs
-**/risc_zero_fibonacci.proof
-**/fibonacci_id.txt
```
Comment (taturosati) [inline scripts/test_examples/risc_zero/fibonacci_proof_generator/.gitignore]: missing newline

File: operator/halo2kzg/halo2kzg_test.go [hunk 2]
```diff
@@ -71,12 +71,12 @@ func TestHalo2KzgProofVerifies(t *testing.T) {
...
-		([halo2kzg.MaxProofSize]byte)(proofBytes), uint32(nReadProofBytes), 
+		([halo2kzg.MaxProofSize]byte)(proofBytes), uint32(nReadProofBytes),
 		([halo2kzg.MaxConstraintSystemSize]byte)(csBytes), uint32(csLen),
 		([halo2kzg.MaxVerifierKeySize]byte)(vkBytes), uint32(vkLen),
 		([halo2kzg.MaxKzgParamsSize]byte)(kzgParamsBytes), uint32(kzgParamsLen),
 		([halo2kzg.MaxPublicInputSize]byte)(publicInputBytes), uint32(nReadPublicInputBytes),
 	) {
 		t.Errorf("proof did not verify")
 	}
-}
```
Comment (taturosati) [inline operator/halo2kzg/halo2kzg_test.go:82]: missing newline

File: operator/halo2ipa/halo2ipa_test.go [hunk 3]
```diff
@@ -71,12 +71,12 @@ func TestHalo2IpaProofVerifies(t *testing.T) {
...
-		([halo2ipa.MaxProofSize]byte)(proofBytes), uint32(nReadProofBytes), 
+		([halo2ipa.MaxProofSize]byte)(proofBytes), uint32(nReadProofBytes),
 		([halo2ipa.MaxConstraintSystemSize]byte)(csBytes), uint32(csLen),
 		([halo2ipa.MaxVerifierKeySize]byte)(vkBytes), uint32(vkLen),
 		([halo2ipa.MaxIpaParamsSize]byte)(ipaParamsBytes), uint32(ipaParamsLen),
 		([halo2ipa.MaxPublicInputSize]byte)(publicInputBytes), uint32(nReadPublicInputBytes),
 	) {
 		t.Errorf("proof did not verify")
 	}
-}
```
Comment (taturosati) [inline operator/halo2ipa/halo2ipa_test.go:82]: missing newline

**Review by taturosati — APPROVED** [review]
finally

**Review by MauroToscano — APPROVED** [review]

**Pull request merged**